deploy/docker/tachyon-dev/tachyon-base/files/log4j.properties

```diff
@@ -19,6 +19,17 @@ log4j.appender.MASTER_LOGGER.layout=org.apache.log4j.PatternLayout
 log4j.appender.MASTER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M) - %m%n
 #log4j.appender.MASTER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M(%L)) - %m%n
 
+# Appender for Master Access Log
```

Comment:
For docker deployment, we are keeping logs in `${tachyon.home}/logs` and for project logging, we are keeping to `${tachyon.logs.dir}`. I don't think we have to make them different, but whoever writes these docker deployment scripts might have better reasons.